src/main/java/com/zendesk/maxwell/MaxwellMysqlConfig.java

```diff
@@ -35,9 +42,18 @@ public void parseOptions( String prefix, OptionSet options) {
 			this.user = (String) options.valueOf(prefix + "user");
 		if ( options.has(prefix + "port"))
 			this.port = Integer.valueOf((String) options.valueOf(prefix + "port"));
+		if ( options.has(prefix + "jdbc_options") ) {
+			String[] opts = ((String) options.valueOf(prefix + "jdbc_options")).replaceAll("\\s+", "").split(",");
```

Comment:
we should only replace leading and trailing whitespace.   Also: probably we should leave the options ampersand separated